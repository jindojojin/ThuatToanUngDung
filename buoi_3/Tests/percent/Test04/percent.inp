4
20 40 20 20 0
0 0 100 0 0
26 0 20 20 34
0 20 12 28 40
