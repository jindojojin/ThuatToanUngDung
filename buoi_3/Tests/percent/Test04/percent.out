5
1
50
25
